pnpm-workspace.yaml

```
@@ -13,7 +13,23 @@ catalogs:
     "effect": ^3.16.3
+  
+  # AI and ML packages
+  ai:
+    "@ai-sdk/assemblyai": 1.0.1
+    "@ai-sdk/deepgram": 1.0.1
+    "@ai-sdk/elevenlabs": 1.0.1
+    "@ai-sdk/gladia": 1.0.1
+    "@ai-sdk/groq": 2.0.4
+    "@ai-sdk/openai": 2.0.7
+    "ai": 5.0.8
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)